### PR TITLE
[php] use latest release

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Use the latest beta version that contains the fix for https://github.com/elastic/apm-integration-testing/pull/972

## Why is it important?

Fix broken tests

## Tests

image

### CODE REVIEW

File: tests/versions/php.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 PHP_AGENT:
   - 'github;master'
-  - 'release;0.2'
+  - 'release;1.0.0-beta1'
```
Comment (SergeyKleyman) [inline tests/versions/php.yml:3]: Maybe we can use `release;latest` instead?

**Review by SergeyKleyman — APPROVED** [review]

File: docker/php/apache/run.sh [hunk 2]
```diff
@@ -15,8 +15,16 @@ if [ -z "${PHP_AGENT_VERSION}" ] ; then
   make clean install
   cp /src/ext-elastic-apm.ini /usr/local/etc/php/conf.d/ext-elastic-apm.ini
 else
-  ## Install the given release
-  wget -q "https://github.com/elastic/apm-agent-php/releases/download/v${PHP_AGENT_VERSION}/apm-agent-php_${PHP_AGENT_VERSION}_all.deb" \
+  if [ "${PHP_AGENT_VERSION}" == "latest" ]  ; then
+    wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+    chmod 755 jq
+    TAG_NAME=$(curl -s https://api.github.com/repos/elastic/apm-agent-php/releases/latest | jq -r .tag_name)
```
Comment (cachedout) [inline docker/php/apache/run.sh:21]:
```suggestion
    TAG_NAME=$(curl -s https://api.github.com/repos/elastic/apm-agent-php/releases/latest | ./jq -r .tag_name)
```
I'm surprised this doesn't need a `./` here if it's downloaded into the current dir. Are you expecting to already be in a directory that's in the $PATH?

Comment (v1v) [reply docker/php/apache/run.sh:21]: `[2020-11-16T12:50:23.753Z] ./run.sh: line 21: jq: command not found` but the shell doesn't stop afterwards :/

**Review by cachedout — APPROVED** [review]

**Comment by v1v** [comment]
@Mergifyio backport 7.x

**Pull request merged**